Pipfile

```diff
@@ -17,3 +17,6 @@ pandas = "<1.5"
 moto = "~=4.1"
 datacompy = "~=0.8"
 docker = "~=6.1"
+pyhive = "~=0.7"
+thrift = "~=0.16"
+thrift-sasl = "~0.4.3"
```

Comment:
Can you speak to what these packages do? 

What Python versions are supported via these packages?

Reply:
My bad, I did not mean to push these. These are unused.